src/components/Form/Form.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+function Form() {
+  return (
+    <form action="#" className="form">
+      <div className="form-group">
+        <label htmlFor="title" className="form-label">
+          <input type="text" name="title" id="title" className="form-input" placeholder="Title.." />
+        </label>
+      </div>
+      <div className="form-group">
+        <label htmlFor="author" className="form-label">
+          <input
+            type="text"
+            name="author"
+            id="author"
+            className="form-input"
+            placeholder="Author.."
+          />
+        </label>
+      </div>
```

Comment:
In line with the previous review, I recommend adding a button to the form in order to enable a user to submit their inputs when they are done

Reply:
@DuaneDave Thanks! Done.